ADVANCED.md

```diff
@@ -653,6 +654,64 @@ We have more discussion and examples [in our issue here](https://github.com/sw-y
 
 </details>
 
+# Section 4: @types/react and @types/react-dom APIs
+
+The `@types` typings export both "public" types meant for your use as well as "private" types that are for internal use.
+
+## `@types/react`
+
+[Link to `.d.ts`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts)
+
+**Namespace: React**
+
+Most Commonly Used Interfaces and Types
+
+- `ReactNode` - anything that is renderable *inside* of JSX, this is NOT the same as what can be returned from a component!
```

Comment:
I guess this used to be `JSXElement`. [`master`](https://github.com/sw-yx/react-typescript-cheatsheet#useful-react-prop-type-examples) currently says otherwise:
> but React.ReactNode is the set of all possible return values of a component.

Maybe switch from `returned from` to `rendered by`? It's IMO slightly ambiguous in its current form.

Reply:
yes i edited it. ok.